.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "lib/dss-interfaces"]
 	path = lib/dss-interfaces
 	url = https://github.com/makerdao/dss-interfaces
+[submodule "lib/dss-exec-lib"]
+	path = lib/dss-exec-lib
+	url = https://github.com/makerdao/dss-exec-lib
```

Comment:
I think that gets specified in the actual reference, no? If not then feel free to tag the release. All of these libraries require very specific commits to prevent these remapping errors.